coordinator/src/orderbook/trading.rs

```diff
@@ -291,6 +302,30 @@ pub async fn process_new_market_order(
         orders::set_order_state(&mut conn, match_param.filled_with.order_id, order_state)?;
     }
 
+    if node.inner.is_connected(order.trader_id) {
+        tracing::info!(trader_id = %order.trader_id, order_id = %order.id, "Executing trade for match");
+        let trade_executor = TradeExecutor::new(node.clone(), notifier);
+
+        tokio::spawn(async move {
+            trade_executor
+                .execute(&TradeAndChannelParams {
+                    trade_params: TradeParams {
+                        pubkey: order.trader_id,
+                        contract_symbol: ContractSymbol::BtcUsd,
+                        leverage: order.leverage,
+                        quantity: order.quantity.to_f32().expect("to fit into f32"),
+                        direction: order.direction,
+                        filled_with: matched_orders.taker_match.filled_with,
+                    },
+                    trader_reserve: channel_opening_params.map(|p| p.trader_reserve),
+                    coordinator_reserve: channel_opening_params.map(|p| p.coordinator_reserve),
+                })
+                .await;
+        });
+    } else {
+        tracing::warn!(trader_id = %order.trader_id, order_id = %order.id, "Skipping trade execution as trader is not connected");
```

Comment:
:+1: I think this warning is okay because we would normally expect the trade to be executed right away.

Reply:
Well, now that you mention it. We would most likely print that warning every time we post an async order.

Should we change it to info?

Reply:
I opted to print a warning on an manual order and an info on an expired order